packages/webapp/pages/account/notifications.tsx

```diff
@@ -585,5 +593,6 @@ const AccountNotificationsPage = (): ReactElement => {
 };
 
 AccountNotificationsPage.getLayout = getAccountLayout;
+AccountNotificationsPage.layoutProps = { seo };
```

Comment:
Only thing that comes to mind is if this needs to be dynamic maybe it would be better as a prop on a layout component, so it's not statically defined and will not be reactive, but not a blocker for this. We can refactor and inject always from one of the `getLayout` wrappers.

Reply:
I tried it at first there actually. Unfortunately, for some reason, it is delayed for a bit and not working as intended. I think it is there on the first few commits of this branch.